make/appimage.mk

```diff
@@ -14,7 +14,15 @@ plugins/timesync.koplugin
 $(filter-out tools/trace_require.lua tools/wbuilder.lua,$(wildcard tools/*))
 endef
 
-update: all
+appimagetool $(APPIMAGETOOL):
+	wget -O $(APPIMAGETOOL).part '$(APPIMAGETOOL_URL)'
+	# Zero-out AppImage magic bytes from the ELF header extended ABI version so
+	# binfmt+qemu can be used (e.g. when executed from `docker run --platform …`).
```

Comment:
Could you add a reference to `https://github.com/AppImage/AppImageKit/issues/1056` in the comment?